database/migration/V20161226_0927__createAttendees.sql

```diff
@@ -115,3 +115,16 @@ CREATE TABLE `attendee_project_interests` (
 		ON DELETE NO ACTION
 		ON UPDATE NO ACTION
 );
+
+CREATE TABLE `attendee_rsvps` (
+	`id` INT UNSIGNED NOT NULL AUTO_INCREMENT,
+	`attendee_id` INT UNSIGNED NOT NULL,
+	`attendee_attendance` ENUM('YES', 'NO', 'YES_TO_CREATE') NOT NULL,
```

Comment:
1. Let's not call this `attendee_attendance`. I know that'll mean you have to rename some stuff, but it's redundant considering that we have it in a table called `attendee_rsvps`
2. Split this off into `attending` (a single bit/boolean value) and `subject` (an enum of either `CREATE` or `CONTRIBUTE`).